lib/routes/account.js

```diff
@@ -946,10 +1013,33 @@ module.exports = function (
         }
 
         function createResponse() {
-          return {
-             email: sessionToken.email,
-             verified: sessionToken.emailVerified
-           }
+
+          var doSigninConfirmation = requestHelper.shouldEnableSigninConfirmation(sessionToken.uid.toString('hex'), config)
+
+          if (doSigninConfirmation) {
+            return db.sessionTokenWithVerificationStatus(sessionToken.id)
```

Comment:
Oh, good point @vbudhram. Maybe there is a case for two separate strategies, `sessionTokenWithVerificationStatus` and `sessionTokenWithVerificationStatusStrict` (or something, that name is off the top of my head)? So the basic strategy would be created via `makeCredentialsFn` and the strict one would be created via `makeVerifiedCredentialsFn`.

Reasonable suggestion or is there a simpler solution?